src/libs/TransactionRecoveryApi/AxelarGMPRecoveryAPI.ts

```diff
@@ -102,8 +102,9 @@ export class AxelarGMPRecoveryAPI extends AxelarRecoveryApi {
    * @returns Promise<boolean> - true if transaction is already executed
    */
   public async isExecuted(txHash: string): Promise<boolean> {
-    const txStatus = await this.queryTransactionStatus(txHash).catch(() => undefined);
-    return txStatus?.status === GMPStatus.EXECUTED;
+    const txStatus: GMPStatusResponse = await this.queryTransactionStatus(txHash);
```

Comment:
Removing catch block here will cause `addNativeGas` and `addGas` function throw unexpected error if the api service is down or whatever fails reason. So we should add it back?

Reply:
### ok, new code block:

`  public async isExecuted(txHash: string): Promise<boolean | undefined> {
    const txStatus: GMPStatusResponse | undefined = await this.queryTransactionStatus(txHash).catch(() => undefined);
    if (!txStatus) return undefined;
    const { status } = txStatus;
    return status !== "error_fetching_status" && status.destExecuted;
  }`